### DISCUSSION TITLE
Reply in PR code review loses scroll position

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#53960] [Mobile] [answered]
GitHub Mobile beta

After I submit a reply on the [PR I'm addressing](https://github.com/ceph/ceph/pull/51230#pullrequestreview-1405127690), GitHub Mobile scrolls me back to the top. This is a real problem as it's quite long (for a mobile device).

**Comment by taki-on** [comment, marked as answer]
Thank you very much for your report! I see it will be painful that it scrolls back when a PR has such a long timeline. We will discuss how to improve this internally.
